README.md

```diff
@@ -49,32 +49,28 @@ OpenDevin agents collaborate with human developers to write code, fix bugs, and
 The easiest way to run OpenDevin is inside a Docker container. It works best with the most recent version of Docker, `26.0.0`.
 You must be using Linux, Mac OS, or WSL on Windows.
 
-To start the app, run these commands, replacing `$(pwd)/workspace` with the directory you want OpenDevin to work with.
+To try OpenDevin in a docker container, run the following Terminal command:
 
 > [!WARNING]
-> OpenDevin runs bash commands within a Docker sandbox, so it should not affect your machine.
-> But your workspace directory will be attached to that sandbox, and files in the directory may be modified or deleted.
+> When you run the following command, files in `./workspace` may be modified or deleted.
 
 ```bash
-# The directory you want OpenDevin to work with. MUST be an absolute path!
-export WORKSPACE_BASE=$(pwd)/workspace;
-```
-
-```bash
-docker run -it \
+mkdir -p $(pwd)/workspace && docker run -it \
     --pull=always \
     -e SANDBOX_USER_ID=$(id -u) \
     -e PERSIST_SANDBOX="true" \
     -e SSH_PASSWORD="make something up here" \
-    -e WORKSPACE_MOUNT_PATH=$WORKSPACE_BASE \
-    -v $WORKSPACE_BASE:/opt/workspace_base \
+    -e WORKSPACE_MOUNT_PATH=$(pwd)/workspace \
+    -v $(pwd)/workspace:/opt/workspace_base \
```

Comment:
I prefer we don't repeat ourselves.

Aside, I agree the original instructions makes you have to copy & paste two commands, but the intention of putting `export ...` in a separate line is:
1) people would pay more attention to it.
2) People could put it in their shell config file.

Reply:
FWIW a number of users are making a `run.sh` with these from the beginning.

Reply:
Re:1, agreed. Can probably do a one liner with an env variable export
2: i think this makes sense from an OpenDevin developer perspective, but people who would need the instructions from the readme probably won't want to edit their bashrc before they could get started with opendevin. Hell, I didn't even want to create a separate work folder.

The new user workflow I am imagining looks something like,

"I have some code i wanna work on and there's this cool tool these guys I don't know from the internet made. Better clone another copy. Oh neat there are these instructions I can copy+paste to get started with!"

Reply:
> I prefer we don't repeat ourselves.
> 
> Aside, I agree the original instructions makes you have to copy & paste two commands, but the intention of putting `export ...` in a separate line is:
> 
> 1. people would pay more attention to it.

agreed. How do you feel about this new edit? It:
- shows the workspace folder up front
- puts things in the same code box such that people can copy it in one go
- uses `&&` to connect things such that e.g. if the workspace folder creation fails, no other commands are run and the failure would not be obscured

> 2. People could put it in their shell config file.

I think this makes sense from an OpenDevin developer perspective, but people who would need the instructions from the readme probably won't want to edit their bashrc before they could get started with opendevin. Hell, I didn't even want to create a separate work folder.

The new user workflow I was imagining looks something like,

"I have some code i wanna work on and there's this cool tool these guys I don't know from the internet made. Better clone another copy. Oh neat there are these instructions I can copy+paste to get started with!"

Reply:
> FWIW a number of users are making a `run.sh` with these from the beginning.

That's kindda neat but also I prefer a blob that i can see & copy to get started. A run.sh feels more appropriate for a repo that I have to clone in order to use.

Reply:
I like the latest edit :)